content/attacks/posts/2011-10-05-Bitcoin7.md

```diff
@@ -0,0 +1,30 @@
+---
+date: 2011-10-05
+target-entities: Bitcoin7
+entity-types:
+  - Exchange
+  - Custodian
+attack-types: Infrastructure Attack
```

Comment:
Add `Wallet Hack` as an attack type, and replace `Infrastructure Attack` with `Web Infrastructure Attack`

Reply:
What's a Web Infrastructure Attack and how is it different from an Infrastructure Attack?

Reply:
Technically it's the same. But there is defined attack types in our wiki, to prevent duplicates, we strict attack types and other headers to match previous ones. If you look through [readme](https://github.com/1712n/dn-institute/tree/main#readme) there are full list of pre-defined types of attacks.